vapi/esx/settings/clusters/vms/list.go

```diff
@@ -67,6 +68,13 @@ type ClusterSolutionInfo struct {
 	//
 	// If unset no AlternativeVmSpecs applied to the System VMs.
 	//Optional<List<AlternativeVmSpec>> alternativeVmSpecs;
```

Comment:
Not part of this change, but this commented out section should be removed in the future please.